src/main.rs

```diff
@@ -157,4 +160,30 @@ fn main() {
         }
         // std::thread::sleep(::std::time::Duration::new(0, 1_000_000_000u32 / 60));
     }
+    log::info!("quitting");
+}
+
+#[cfg(debug_assertions)]
+fn init_logger() {
+    use log::LevelFilter;
+    use simplelog::{ColorChoice, Config, TermLogger, TerminalMode};
+
+    TermLogger::init(
+        LevelFilter::Debug,
+        Config::default(),
+        TerminalMode::Mixed,
+        ColorChoice::Auto,
+    )
+    .expect("cannot initialise logger");
+}
+
+#[cfg(not(debug_assertions))]
+fn init_logger() {
+    use log::LevelFilter;
+    use simplelog::{Config, WriteLogger};
+    use std::fs::File;
+
+    let file = File::create("/tmp/gbmu.log").expect("cannot open log file");
```

Comment:
are we sure `/tmp/gbmu.log`will always be available to put our logfile ? maybe the folder of the app might be more reliable ?

Reply:
We cannot be sure the `/tmp/gbmu.log` to be always available.
We have the same problem if we use the current app folder (the folder could read-only).